### PR TITLE
Enable more doctests in Series

### PR DESCRIPTION
**Pull request by ueshin**
Enable doctests in `series.py` and make `run-tests.sh` handle doctest failures properly.

### CODE REVIEW

File: dev/run-tests.sh [hunk 1]
```diff
@@ -34,6 +34,4 @@ else
     ARGS="$@"
 fi
 # Runs the main program that finds and runs the docstrings.
-exec python databricks/koalas/testing/doctest_main.py
-# Then run the test suites
-exec nosetests $ARGS --where "$FWDIR"
+python databricks/koalas/testing/doctest_main.py && nosetests $ARGS --where "$FWDIR"
```
Comment (HyukjinKwon) [inline dev/run-tests.sh]: cool

File: databricks/koalas/series.py [hunk 2]
```diff
@@ -166,16 +166,16 @@ def dtype(self):
 
         Examples
         --------
-        >>> s = ks.Series([1, 2, 3])  # doctest: +SKIP
-        >>> s.dtype  # doctest: +SKIP
-        dtype('int64')
+        >>> s = ks.Series([1, 2, 3])
+        >>> s.dtype
+        dtype('int64')a
```
Comment (HyukjinKwon) [inline databricks/koalas/series.py]: was this `a` type :)?

Comment (ueshin) [reply databricks/koalas/series.py]: That's on purpose to see how travis-ci works. Will revert it later.

Comment (ueshin) [reply databricks/koalas/series.py]: Travis-ci reported failure as expected. revert it.

**Comment by rxin** [comment]
This is great.

**Pull request merged**